test/tools/NamedPipeConnection/src/code/Microsoft.PowerShell.NamedPipeConnection.csproj

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.PowerShell.NamedPipeConnection</RootNamespace>
+    <AssemblyName>Microsoft.PowerShell.NamedPipeConnection</AssemblyName>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <FileVersion>1.0.0</FileVersion>
+    <InformationalVersion>1.0.0</InformationalVersion>
+    <TargetFrameworks>net7.0</TargetFrameworks>
+    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+    <LangVersion>9.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" version="4.5.0-*" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
+    <PackageReference Include="System.Management.Automation" Version="7.*" />
```

Comment:
Can this just be a project reference?

Reply:
This feature uses new APIs that are not in any package reference AFAIK.  But at some point, once the APIs stabilize, we can use a package reference.

Reply:
Will use project reference instead.